### ISSUE TITLE
TypeError: estree.clearProgramCache is not a function

### ISSUE THREAD

**Issue by sovcik** [microsoft/DefinitelyTyped-tools#621]
Steps to reproduce:
1. clone https://github.com/DefinitelyTyped/DefinitelyTyped
2. run `npm test node`

Error:
```
dtslint@0.0.148
testing from 4.2 to 4.8 in /home/jozef/DefinitelyTyped/types/node/ts4.8
TypeError: estree.clearProgramCache is not a function
    at lint (/home/jozef/DefinitelyTyped/node_modules/@definitelytyped/dtslint/src/lint.ts:79:12)
    at testTypesVersion (/home/jozef/DefinitelyTyped/node_modules/@definitelytyped/dtslint/src/index.ts:233:15)
    at runTests (/home/jozef/DefinitelyTyped/node_modules/@definitelytyped/dtslint/src/index.ts:197:7)
    at main (/home/jozef/DefinitelyTyped/node_modules/@definitelytyped/dtslint/src/index.ts:83:5)
```

**Comment by sovcik** [comment]
Issue does not occur with dtslint@0.0.147

**Comment by sovcik** [comment]
fixed by #620
